mycroft/audio/interface.py

```diff
@@ -102,68 +121,38 @@ def play(self, tracks=None, utterance=None, repeat=None):
             tracks = [tracks]
         elif not isinstance(tracks, list):
             raise ValueError
-        tracks = [ensure_uri(t) for t in tracks]
-        msg = self._format_msg('mycroft.audio.service.play',
```

Comment:
Same as above, any possibility for third party integrations or skills needing this?